ox-chameleon.el

```diff
@@ -103,7 +118,131 @@ When set to nil, the current theme will be used.")
                         (if (and (< lf 0.4) (< (* sf lf) 0.1)) "#000000" fg)))
               (list bg fg)))))
 
-(defun ox-chameleon--generate-text-colourings ()
+(defun ox-chameleon--symbol->weight (weight)
+  (pcase weight
+    ('ultra-light 100) ('extra-light 100)
+    ('light 200) ('thin 200)
+    ('semi-light 300)
+    ('book 400) ('normal 400) ('regular 400)
```

Comment:
Did you get these values from somewhere? I'd be inclined to set:
- ultra-light/extra-light: 100
- thin: 200
- light: 300
- semi-light: 350
- normal/regular: 400
- book: 450

Reply:
engrave-faces-html :p

Reply:
Where did I get them from... :thinking:

Reply:
Ok, I've revisited the values in engrave-faces and used the OpenType spec to pick something a bit better.

Why don't you just call `engrave-faces-html--css-weight`? This looks like pointless duplication.